stock_move_line_change_lot/models/stock_move_line.py

```diff
@@ -85,7 +85,7 @@ def write(self, vals):
                 vals.get("location_id", move_line.location_id.id)
             )
             package = move_line.package_id.browse(
-                vals.get("package_id", move_line.package_id)
+                vals.get("package_id", move_line.package_id.id)
             )
```

Comment:
I would even add a call to `exists()` to filter out wrong ID in user's input
```suggestion
            ).exists()
```